packages/legacy/src/common/constants.ts

```diff
@@ -53,7 +53,7 @@ export enum StatusCategoryType {
   Critical = 'Critical',
   Error = 'Error',
   Advisory = 'Advisory',
-  Warn = 'Warn',
+  Warn = 'Warning',
```

Comment:
IMHO this should match `github.com/konveyor/controller/pkg/condition`

Reply:
check https://github.com/kubev2v/forklift-controller/blob/e73b65a01cda13890538c57f1cda2261f206d7f3/pkg/controller/plan/validation.go#L63

Reply:
ok, the fix should be in the backend :-)
I'll leave it as the backed, and add a task to fix it in the backend :-)
IMHO `warn` is not a god category name :-)